client/flutter/android/app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "1057717871229",
+    "firebase_url": "https://who-myhealth-staging.firebaseio.com",
+    "project_id": "who-myhealth-staging",
+    "storage_bucket": "who-myhealth-staging.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:1057717871229:android:4fbe832a3de230450dc714",
+        "android_client_info": {
+          "package_name": "org.who.WHOMyHealth"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "1057717871229-i95ol1e2smafmpja9ujf3vt3ru4hmcqh.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyALhiQKpx8jjtC6OIyaj3cJVTp8xDrcSnY"
+        }
+      ],
+      "services": {
+        "appinvite_service": {
+          "other_platform_oauth_client": [
+            {
+              "client_id": "1057717871229-i95ol1e2smafmpja9ujf3vt3ru4hmcqh.apps.googleusercontent.com",
+              "client_type": 3
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "configuration_version": "1"
+}
```

Comment:
API keys and client IDs should usually be hidden from public/open source projects

I'm not sure the best solution, because you can store this as B64 in an env variable for the CI, but contributors also need access to it

Reply:
It's fine to do this for Firebase -- see https://groups.google.com/forum/#!msg/firebase-talk/bamCgTDajkw/uVEJXjtiBwAJ

Reply:
So the same API keys are used for other Google Services (including Maps/Places/Push Notifications). But as the post you linked says: if someone wanted to abuse these credentials they'd not need this JSON (as it's embedded in the app package and can be intercepted when API requests are made). Also as these are staging credentials it's not like anything

I'm more highlighting general "good practice" guidance of not publishing API keys to the public.

One solution is to skip the Google Play Services plugin in the Gradle build file when a google-services.json file isn't found (see https://stackoverflow.com/a/49304685/2233621) and that way the build shouldn't fail.

Reply:
@epicfaace the linked post says "For a library or open-source sample we do not include the JSON file because the intention is that users insert their own to point the code to their own backend.  That's why you won't see JSON files in most of our firebase repos on github."

Reply:
@advayDev1 this file is the staging config for now. do you have concerns with checking that one into the repo?

Reply:
checking with bob